index.html

```diff
@@ -39,19 +39,42 @@
         <h2>Controls</h2>
         <h3>Date Range</h3>
         <form name="dateSelection">
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioMonth" checked>
             <label for="radioMonth">Past 30 Days</label>
           </div>
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioWeek">
             <label for="radioWeek">Past 7 Days</label>
           </div>
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioDay">
             <label for="radioDay">Today</label>
           </div>
         </form>
+        <h3>Data Filter</h3>
```

Comment:
Likewise here: do we need a title? If so, I think it should at least be "Legend" instead